pyoverkiz/client.py

```diff
@@ -147,6 +165,16 @@ async def login(
         Authenticate and create an API session allowing access to the other operations.
         Caller must provide one of [userId+userPassword, userId+ssoToken, accessToken, jwt]
         """
+        # Local authentication
+        if self._is_local:
+            if register_event_listener:
+                await self.register_event_listener()
+            else:
+                # Call a simple endpoint to verify if our token is correct
+                await self.get_api_version()
```

Comment:
This API endpoint doesn't have authentication 🤦‍♂️. I need to find another simple endpoint to verify a successful login.